workflows/ovp/create_and_manage_youtube_videos_as_native_kaltura_external_media_entry_/readme.md

```diff
@@ -0,0 +1,181 @@
+<!--METADATA
```

Comment:
@tzubeli - this pull seems to include an addition Zohar made in a different one and one that is unrelated to the matter at hand. Please remove this file from this branch.